### PR TITLE
Dat 1210 create doppler usage system service

### PR DESCRIPTION
**Pull request by jcamposmk**
### **Create doppler usage system service**
**Jira Ticket:** https://makingsense.atlassian.net/browse/DAT-1210

**Description**

Create doppler system usage client

### CODE REVIEW

File: src/services/doppler-system-usage-api-client.double.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { ResultWithoutExpectedErrors } from '../doppler-types';
+import { timeout } from '../utils';
+import { SystemUsage } from './doppler-system-usage-api-client';
+
+export interface DopplerSystemUsageApiClient {
+  getUser(): Promise<ResultWithoutExpectedErrors<SystemUsage>>;
```
Comment (andresmoschini) [inline src/services/doppler-system-usage-api-client.double.ts:6]: What do you think about it?

```suggestion
  getUserSystemUsage(): Promise<ResultWithoutExpectedErrors<SystemUsage>>;
```

File: src/services/doppler-system-usage-api-client.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+export class HttpDopplerSystemUsageApiClient
+  extends HttpBaseClient
+  implements DopplerSystemUsageApiClient
+{
+  private clientName = 'System Usage';
+
+  public async getUser(): Promise<ResultWithoutExpectedErrors<SystemUsage>> {
+    try {
+      const { email, jwtToken } = this.getApiConnectionData(this.clientName);
```
Comment (andresmoschini) [inline src/services/doppler-system-usage-api-client.ts:21]: What is the meaning of getApiConnectionData's parameter?

File: src/services/doppler-system-usage-api-client.ts [hunk 3]
```diff
@@ -0,0 +1,44 @@
+import { HttpBaseClient } from './http-base-client';
+import { ResultWithoutExpectedErrors } from '../doppler-types';
+
+export interface SystemUsage {
+  email: string;
+  reportsSectionLastVisit: string;
```
Comment (andresmoschini) [inline src/services/doppler-system-usage-api-client.ts:6]:
```suggestion
  reportsSectionLastVisit: string | undefined;
```

**Comment by andresmoschini** [comment]
Nice!

File: src/services/doppler-system-usage-api-client.ts (hunk 2 above)
Comment (jcamposmk) [reply src/services/doppler-system-usage-api-client.ts:21]: This parameter is only used to print who was the client that generated an error ... 
you can see its usage in src/services/http-bas
Captura de Pantalla 2022-10-28 a la(s) 9 03 06 a  m
e-client.ts

Comment (andresmoschini) [reply src/services/doppler-system-usage-api-client.ts:21]: 👍

**Pull request merged**